contrib/sequoia-sq/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "sequoia-sq"
+pkgver = "0.34.0"
+pkgrel = 0
+build_style = "cargo"
+make_build_env = {"ASSET_OUT_DIR": "assets"}
+_deps = ["nettle-devel", "openssl-devel", "sqlite-devel", "capnproto-devel"]
+hostmakedepends = list(_deps + ["cargo", "pkgconf", "capnproto"])
+makedepends = list(_deps + ["rust-std"])
```

Comment:
i think you only need the -devels in make and not host? but i guess you tested cross so that's why

you don't need the list() there

Reply:
There's some questionable choices in there that result in the sequoia-openpgp crate being built for both the host and the target. This means the dependencies of said crate are required for both host and target, which is why they are needed for both when cross compiling.

Reply:
I've removed the `list()` calls but kept the rest as is.